src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -15,28 +19,19 @@ import { getFilterFromPath } from "../../../Core/getFilterFromPath.service";
 import { generatePathString } from "../../../Core/generatePathString.service";
 import { getFeaturesFromPath } from "../../../Core/getTableFromPath.service";
 
-interface FilterConfigApiInterface {
-    filters: {
-        id: string;
-        name: string;
-        parent: string;
-        values: string[];
-    }[];
-}
-
 export function QueryPageContainerComponent(): JSX.Element {
     const { data, setData } = useContext(DataContext);
     const { filter, setFilter } = useContext(FilterContext);
     const { table, setTable } = useContext(TableContext);
     const history = useHistory();
 
-    const BASE_URL: string = mockDataURL;
+    const BASE_URL: string = isolateURL;
     const FILTER_URL: string = filterURL;
 
     const getData = async (): Promise<void> => {
-        const r: Response = await fetch(BASE_URL);
-        const dataProp: DBentry[] = await r.json();
-        const keyValueProps = Object.keys(dataProp[0]) as DBkey[];
+        const isolateResponse: Response = await fetch(BASE_URL);
+        const isolateProp: IsolateApiInterface = await isolateResponse.json();
+        const keyValueProps = Object.keys(isolateProp.isolates[0]) as DBkey[];

```

Comment:
As stated in a previous review, consider refactoring this. Your data should not depend on the first entry of an api response. The client should know what it expects from the api and the data format should be clear for both sides, client and server.

Reply:
Done. 
- Only use desired values from isolate-API